app/assets/stylesheets/button.scss

```diff
@@ -0,0 +1,10 @@
+.govuk-button {
```

Comment:
If we're overriding all the css - it'd be worth just creating `.dfe-button` for this rather than overriding all the CSS with `!important`

Reply:
We don't override all of the CSS of the `.govuk-button`, just some of them. The `govuk-frontend` does a lot of styling behind the scenes. So I don't think it's worth atm to create a new CSS class for it.